index.html

```diff
@@ -7,14 +7,202 @@
+        <div class="row">
+            <div class="col-md-6 order-md-5 g-0">
+                <img class = "img-fluid" id="flower" src = "./img/green-flower.png" alt = "Green-Blue Flower">
+            </div>
+            <div class="col-md-6 order-md-1 g-0">
+                <div class = introduction>
+                    <h1 class = "page-title">
+                        Online Therapy<br>for Emotional<br>Health
+                    </h1>
+                    <!--Intro-->
+                    <p class = "description">
+                        I am a description. Click here to edit.
+                    </p>
+                    <!--Scroll down arrow-->
+                    <a href = "#">&darr;</a>
+                </div>
+            </div>
+        </div>
+    </div>
+    <div class = "container text-center g-0">
+        <div class = "row">
+            <div class = "col g-0">
+                <!--About Section-->
+                <section class = "about-maria">
+                    <h3 class = "About">About</h3>
+                    <h2 class = "Psych-name">Maria Hope</h2>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click "Edit Text" or double click me to add your own content and make changes to the font.
+                        Feel free to drag and drop me anywhere you like on your page. I'm a great place for you to tell a story and let your users know a little more about you.
+                    </p>
+                    <a href = "#">&rarr;</a>
+                </section>
+            </div>
+        </div>
+        <div class = "row">
+            <div class = "col g-0">
+                <img class = "maria-photo img-fluid" src = "./img/maria-hope.png" alt = "Maria Hope">
+            </div>
+        </div>
+    </div>
+    <!--Therapies Section-->
+    <div class = "container text-center g-0">
+        <section class = "therapies">
+        <div class = "row">
+            <div class = "col-md-4 g-0">
+                <section class = "individual">
+                    <h3>01.</h3>
+                    <h2>Individual Therapy</h2>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click "Edit Text" or double click me to add your own content and make changes to the font.
+                    </p>
+                </section>
+            </div>
+            <div class = "col-md-4 g-0">
+                <section class = "group">
+                    <h3>02.</h3>
+                    <h2>Group Therapy</h2>
+                    <p>
+                        I'm a paragraph. Click here to add your own text and edit me. It's easy. Just click "Edit Text" or double click me to add your own content and make changes to the font.
+                    </p>
+                </section>
+            </div>
+            <div class = "col-md-4 g-0">
+                <section class = "couples">
```

Comment:
I notice on desktop these 3 coloured blocks aren't all the same height. That's because they adapt to their content (i.e. less text needs less height). 
The 3 `col-md-4` blocks _are_ all the same height however because they are flex children and flex children always have the same height (their parent `row` has `display: flex`) so you can just set `height: 100%` to the coloured blocks (`individual `, `group` and `couples`) so they will grow to the height of their respective parents (which is `col-md-4`).
Hope that makes sense - happy to chat about this is it's not clear!